demos/package.json

```diff
@@ -4,8 +4,7 @@
   "private": true,
   "scripts": {
     "start": "vite --host",
-    "build": "npm run ts && vite build",
-    "preview": "vite preview",
+    "build:demo": "npm run ts && vite build",
```

Comment:
Should we call it `build:demos` for the sake of consistency?

Reply:
Done!